build/cmake/package_tester/modules/tests.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
```

Comment:
Please include a comment at the top of this file. I'd like to make it more clear that the idea is to write tests in this file that can be reused for any package, and maybe explain the requirements to specialize these tests to a particular package.

Reply:
I added a comment in my last commit